mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -92,10 +96,12 @@ class ExperimentPage extends Component {
   onSearch(paramKeyFilterString, metricKeyFilterString, searchInput, lifecycleFilterInput) {
     const andedExpressions = SearchUtils.parseSearchInput(searchInput);
     this.setState({
-      paramKeyFilterString,
-      metricKeyFilterString,
-      searchInput,
-      lifecycleFilter: lifecycleFilterInput
+      persistedState: {
```

Comment:
Should we pass this through the ImmutableRecord constructor? Just to make it more obvious that this field is effectively typed.